src/kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -505,7 +518,11 @@ impl<'tcx> GotocCtx<'tcx> {
         let loc = self.codegen_caller_span(&span);
         Stmt::block(
             vec![
-                self.codegen_assert_false(msg, loc),
+                self.codegen_assert_false(
+                    PropertyClass::CustomProperty("fatal-error".to_string()),
```

Comment:
I'm not actually sure how `codegen_fatal_error` is used, so I'm not sure what property class this one should be.